external/js/cothority/src/byzcoin/contracts/ro-pa-sci-instance.ts

```diff
@@ -177,25 +177,52 @@ export default class RoPaSciInstance {
 export class RoPaSciStruct extends Message<RoPaSciStruct> {
     readonly description: string;
     readonly stake: Coin;
-    readonly firstplayerhash: Buffer;
-    readonly firstplayer: number;
-    readonly secondplayer: number;
-    readonly secondplayeraccount: Buffer;
+    readonly firstPlayerHash: Buffer;
+    readonly firstPlayer: number;
+    readonly secondPlayer: number;
+    readonly secondPlayerAccount: Buffer;
+
+    constructor(props?: Properties<RoPaSciStruct>) {
+        super(props);
+
+        this.firstPlayerHash = Buffer.from(this.firstPlayerHash || EMPTY_BUFFER);
+        this.secondPlayerAccount = Buffer.from(this.secondPlayerAccount || EMPTY_BUFFER);
+
+        Object.defineProperty(this, "firstplayer", {
```

Comment:
does the capitalization matter? `Object.defineProperty(this, "firstplayer"` vs `this.firstPlayer` and `Object.defineProperty(this, "firstplayerhash",` vs `this.firstPlayerHash`

Reply:
Yes it is important because the defineProperty will create the necessary fields for protobuf to work. Thus yes the _secondPlayer_ is wrong, I will fix it, thanks!

Reply:
It has not been caught because those tests are disabled until personhood is merged.